packages/wonder-blocks-button/components/button.vite_test.js

```diff
@@ -0,0 +1,19 @@
+/* global render */
```

Comment:
That seems annoying - is there a way in which you can make that "just work" like it does with describe/test/etc. coming from jest?

Reply:
That's on my list of things to do.

Reply:
I think we'll have to make our own custom eslint environment.  If `vite` gets popular enough then maybe it will become an official eslint environment like `jest`.  Here's the docs on how to do this: https://eslint.org/docs/developer-guide/working-with-plugins#environments-in-plugins